<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-G7HDG2O" failures="1" tests="1" name="ThreeTest" time="33.854" errors="0" timestamp="2021-12-26T10:51:52 IST">
  <testcase classname="test.ThreeTest" name="testThree" time="28.753">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.ThreeTest.testThree(ThreeTest.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- testThree -->
</testsuite> <!-- ThreeTest -->
